contributed_definitions/NXenergydispersion.nxdl.xml

```diff
@@ -69,19 +69,79 @@
+                     leads to a changing energy resolution (∆E ~ Ep) at different kinetic energies. It can however
+                     also be used in XPS.
+                     
+                     Synonyms: constant ΔE/E mode, constant retardation ratio mode, CRR mode, FRR mode
+                     
+                         This concept is related to term `12.66`_ of the ISO 18115-1:2023 standard.
+                     .. _12.66: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.66
+                </doc>
+            </item>
+            <item value="fixed_energy">
+                <doc>
+                     In the fixed energy (FE) mode, the intensity for one single kinetic energy is measured for a
+                     specified time. This mode is particulary useful during setup or alignment of the
+                     electron analyzer, for analysis of stability of the excitation source or for sample
+                     alignment.
+                     
+                     Since the mode measures intensity as a function of time, the difference in channel signals 
+                     is not of interest. Threfore, the signals from all channels are summed.
+                     
+                     Synonyms: FE mode
+                </doc>
+            </item>
+            <item value="snapshot">
+                <doc>
+                     Snapshot mode does not involve an energy scan and instead collects data from all channels of
+                     the detector without averaging. The resulting spectrum reflects the energy distribution of
+                     particles passing through the analyzer using the current settings. This mode is commonly used
+                     to position the detection energy at the peak of a peak and record the signal, enabling faster
+                     data acquisition within a limited energy range compared to FAT. Snapshot measurements are
+                     particularly suitable for CCD and DLD detectors, which have multiple channels and can accurately
+                     display the peak shape. While five or nine-channel detectors can also be used for snapshot
+                     measurements, their energy resolution is relatively lower.
+                </doc>
+            </item>
+            <item value="transmission">
+                <doc>
+                     MISSING DOC.
+                     
+                         This concept is related to term `&lt;term&gt;`_ of the &lt;spec&gt; standard.
+                     .. _&lt;term&gt;: &lt;url&gt;
+                </doc>
+            </item>
+            <item value="jittered">
+                <doc>
+                     MISSING DOC.
+                     
+                         This concept is related to term `&lt;term&gt;`_ of the &lt;spec&gt; standard.
+                     .. _&lt;term&gt;: &lt;url&gt;
```

Comment:
In dither acquisition mode, the kinetic energy of the analyzer is randomly varied by a small value around a central value and at fixed pass energy. This allows reducing or removing inhomogeneities of the detector efficiency, such as e.g. imposed by a mesh in front of the detector. Mostly relevant for CCD/DLD type of detectors.

Reply:
Thanks, I put it into the docstring